src/index.js

```diff
@@ -1,4 +1,4 @@
-import { cloneElement, h, Component } from 'preact';
+import { cloneElement, createElement, Component, toChildArray } from 'preact';
```

Comment:
We might end up changing the semantics of toChildArray and I'm kinda worried about relying on it here. Could we use a local helper function instead?

```js
// L165:  const children = getChildren([], this.props.children);
function getChildren(accumulator, children) {
	if (Array.isArray(children)) {
		children.reduce(getChildren, accumulator);
	}
	else if (children) {
		accumulator.push(children);
	}
	return accumulator;
}
```

Reply:
Actually it's okay to hold off on this. A fix has been merged upstream.